src/JWT/JwtValidator.cs

```diff
@@ -55,6 +56,48 @@ public void Validate(string payloadJson, string decodedCrypto, string decodedSig
             ValidateNbfClaim(payloadData, secondsSinceEpoch);
         }
 
+        private static bool AreAllDecodedSignaturesNullOrWhiteSpace(string[] decodedSignatures)
+        {
+            return decodedSignatures.All(string.IsNullOrWhiteSpace);

```

Comment:
I'd define lambda variable here too. And please use full type name rather than the alias (string -> String) for consistency across the codebase